BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.m

```diff
@@ -385,8 +384,9 @@ - (id)copyWithZone:(NSZone *)zone
                                                                                           body:bodyCopy
                                                                                    queryParams:queryStringParametersCopy
                                                                                  session:self.session];
+    self.outputStream.delegate = nil;
     operationCopy.outputStream = self.outputStream;
-    operationCopy.outputStream.delegate = nil;
+    self.outputStream = nil;
```

Comment:
This line is *the* bug fix. Shortly after this code executes `[self finish]` is called. And by nil'ing out the output stream, it's left intact for the copy (thus avoiding a crash). Now, it's still the case that moving the outputStream from one operation to another might be semantically incorrect. If that turns out to be the case then this fix for the flakiness/crashes should be to make BOXDataAPIOperation avoid this `copy` code path in the face of a token-expired error.